lib/BitcastUtils.cpp

```diff
@@ -702,4 +702,33 @@ void ConvertInto(Type *Ty, IRBuilder<> &Builder,
   }
 }
 
+bool RemovedCstExprFromFunction(Function *F) {
```

Comment:
Almost every call to this function does it in a loop. I'd suggest making this worklist driven and hitting all the constant expressions at once to simplify the callers.

Reply:
Done